lib/api/controllers/userController.js

```diff
@@ -23,25 +23,20 @@
 
 const { isNil } = require('lodash');
 const Bluebird = require('bluebird');
-const { v4: uuidv4 } = require('uuid');
 
-const { KuzzleError } = require('../../kerror/errors');
 const kerror = require('../../kerror');
-const { has, get } = require('../../util/safeObject');
+const { get } = require('../../util/safeObject');
 const { Request } = require('../request');
-const nameGenerator = require('../../util/name-generator');
 const formatProcessing = require('../../core/auth/formatProcessing');
-const NativeSecurityController = require('./base/nativeSecurityController');
+const AbstractSecurityController = require('./base/abstractSecurityController');
 
 /**
  * @class UserController
  */
-class UserController extends NativeSecurityController {
+class UserController extends AbstractSecurityController {
   constructor() {
     super([
       'create',
```

Comment:
I know it's really nitpicking but can you sort the action names alphabetically ?